get-started/mining/index.md

```diff
@@ -78,8 +78,12 @@ meta_descr: meta_descr.mining
                     <h2>{% t mining.software %}</h2>
                         <p>{% t mining.softwarep %}
                         <ul>
-                            <li><a href="https://github.com/xmrig/xmrig">XMRig</a></li>
-                            <li><a href="https://github.com/cryptonote-social/csminer">CSminer</a></li>
+                            <li><a href="https://gupax.io/">Gupax</a>
```

Comment:
Gupax or gupaxx

Reply:
I don't know how the raffle system exactly works, how xmrvsbeast or the donors benefit from giving free computing powers?

Reply:
I don't either, but XvB has always given away hashrate (since before p2pool).

my gupax vs gupaxx comment isnt re the raffle, but about being maintained.

I havent checked, but i thought gupaxx was created because gupax was unmaintained

Reply:
I don't either, but XvB has always given away hashrate (since before p2pool).

my gupax vs gupaxx comment isnt re the raffle, but about being maintained.

I havent checked, but i thought gupaxx was created because gupax was unmaintained